### ISSUE TITLE
cannot repeat track in albums, playlists

### ISSUE THREAD

**Issue by rbrtbrnschn** [Rigellute/spotify-tui#800]
DISCLAIMER: I believe this to be a spotifyd limitation or well a spotify API limitation, but as far as i am concerned i cannot set it to repeat track (so same song over and over), not even when i am connected to spotifyd on my phone.

on the tui client it just jumps back to All after setting it to track.

Maybe you know if thats something you can fix?

**Comment by sputnick1124** [comment]
You are correct that this is a spotifyd (actually a librespot) [issue](https://github.com/Spotifyd/spotifyd/issues/514). 

See #661 for more discussion on this.

**Comment by johnfromoptus** [comment]
> You are correct that this is a spotifyd (actually a librespot) [issue](https://github.com/Spotifyd/spotifyd/issues/514).
> 
> See #661 for more discussion on this.
> 
> Ultimately, it appears to be because librespot only offers the option of boolean toggle for repeat (see https://github.com/librespot-org/librespot/blob/4925adb4f103cea20f38eee1db9f09b0d3e07e75/protocol/proto/spirc.proto#L94). ~This might actually be a petty simple fix their lib if someone gets around to it.~ Looks like this is being tracked by [librespot-org/librespot#19](https://github.com/librespot-org/librespot/issues/19).

Weird, I came looking for this issue because I recently upgraded spotify-tui and I swear I was previously able to repeat single tracks. Do you know of a workaround?

**Comment by sputnick1124** [comment]
Unfortunately, the only workaround I know of is to use something other than spotifyd as the player device.
